dpypelines/s3_tar_received.py

```diff
@@ -67,12 +67,23 @@ def start(s3_object_name: str):
     # get_dataset_id() currently empty - returns "not-specified"
```

Comment:
Update comment to reflect changes here

Reply:
would it be okay to just delete that 2 lines of comment?